front-end/virtual_pet/lib/pages/pomodoro_page.dart

```diff
@@ -25,14 +25,14 @@ class PomodoroState extends State<PomodoroPage> {
             style: TextStyle(color: Colors.black),
           ),
         ),
-        body: Container(
-          child: const Text(
-            "Pomodoro timer coming soon...",
-            style: TextStyle(
-              color: Colors.black,
-              fontSize: 40.0,
-            ),
+        body:
+        const Text(
```

Comment:
nit: inconsistent indenting between line 20-21 and 28-29

Suggestion:
```
body: const Text(
```

Reply:
does `flutter analyze` not automatically reformat the code @lou-weng, or was it just not run before creating a pull request @wu-jacob

Reply:
I think it got past the linter. The linter flagged the previous commit because this entire section was wrapped with an unnecessary Container() object. 

I don't think the linter checks for this particular formatting issue, else the github pipeline would have flagged it